server/schema/customer/res.js

```diff
@@ -97,6 +102,17 @@ const resolvers = {
         return input
       })
     },
+    async createCustomer(_, {input}, {loggedInUser}) {
+      _authAdmin(loggedInUser)
+      var user = await Customer.findOne({where: {username: input.username, type: 'password'}})
```

Comment:
no need : type: 'password'
coz if there a user of type:'facebook'  -> user if still null even if there is already a fb registered user